internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,60 @@
 package daysteps
 
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
 	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
 	// TODO: реализовать функцию
+	datastr := strings.Split(datastring, ",")
+	if len(datastr) != 2 {
+		return errors.New("mistake with datastring format")
+	}
+
+	steps, err := strconv.Atoi(datastr[0])
+	if steps <= 0 {
+		return errors.New("steps is equal or less than zero")
+	}
+	if err != nil {
+		return fmt.Errorf("mistake with parsing int type :%w", err)
```

Comment:
Отличное оборачивание ошибки.
```suggestion
		return fmt.Errorf("mistake with parsing int type: %w", err)
```